### PR TITLE
Add/v2 history

### PR DESCRIPTION
**Pull request by flapezoti**

### CODE REVIEW

File: app/histories_v2.go [hunk 1]
```diff
@@ -3,13 +3,44 @@ package app
...
+
+		// retrieve messages
+		messages := make([]*models.MessageV2, 0)
+		collection := app.Defaults.MongoMessagesCollection
+
+		for _, topic := range authorizedTopics {
+			topicMessages := mongoclient.GetMessagesV2(c, topic, from, limit, collection)
+			messages = append(messages, topicMessages...)
+		}
+		return c.JSON(http.StatusOK, messages)
```
Comment (lmsilva-wls) [inline app/histories_v2.go:42]: Duplicate

File: app/history_test.go [hunk 2]
```diff
@@ -143,6 +139,9 @@ func TestHistoryHandler(t *testing.T) {
 				var messages []models.Message
 				err = json.Unmarshal([]byte(body), &messages)
 				Expect(err).To(BeNil())
+				print("messages")
+
+				print(messages)
```
Comment (lmsilva-wls) [inline app/history_test.go]: These can be removed, right?

**Review by lmsilva-wls — COMMENTED** [review]
LGTM overall, just some minor details

File: app/histories_v2.go (hunk 1 above)
Comment (flapezoti) [reply app/histories_v2.go:42]: Fixed it

File: app/history_test.go (hunk 2 above)
Comment (flapezoti) [reply app/history_test.go]: Yes, removed as well

**Review by lmsilva-wls — APPROVED** [review]

**Pull request merged**